### ISSUE TITLE
does it surpport Android N？

### ISSUE THREAD

**Issue by zilinhero** [devadvance/rootcloak#182]

**Comment by Crescendo-BLYAT** [comment]
yes, just don't use the Native Library, it'll make your phone crazy....
you'll get random error....

**Comment by eusebiu** [comment]
I used the native library on Android P on a Nexus 5 (had to manually compile armeabi-v7a) but without much success... I guess it did it's job but the app I was testing on was using some other commads or keywords.
